website/src/app/[lang]/[country]/(website)/selection/page.tsx

```diff
@@ -0,0 +1,193 @@
+'use client';
+import { DefaultPageProps } from '@/app/[lang]/[country]';
+import { Disclosure } from '@headlessui/react';
+import { ChevronDownIcon } from '@heroicons/react/20/solid';
+import { Translator } from '@socialincome/shared/src/utils/i18n';
+import { BaseContainer, Typography } from '@socialincome/ui';
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { LoaderIcon } from 'react-hot-toast';
+
+export default function Page(props: DefaultPageProps) {
+	const [translator, setTranslator] = useState<Translator>();
```

Comment:
There is a `useTranslator` hook for this :)

Reply:
But generally, I would argue that it's better to load the translations inside a server component and pass them to the client component (see `login/page.tsx` as an example). By doing that, the statically rendered pages will contain the text. Otherwise, the text will only show after the client components have been hydrated.